src/content/docs/products/splashkit/SplashKit Tutorials/Onboarding/03-pull-request.md

```diff
@@ -0,0 +1,75 @@
+---
+title: How to Create a Pull Request
+description: This is a step-by-step guide on how to create a pull request for SplashKit tutorials.
+sidebar:
+  label: 3. Pull Request Guide
+  order: 3
+---
+
+## How to Create a Pull Request
+
+Follow these steps to create a pull request for the SplashKit tutorials repository.
+
+### 1. Check for Upstream Branches
+
+First, check if the upstream branches are already added to your local repository.
+
+```shell
+git remote -v
+```
+
+### 2. Add Upstream Branches (if not present)
+
+If the upstream branches are not added, you can add them using the following command. Replace `<repo-name>` with the actual repository name.
+
+```shell
+git remote add upstream https://github.com/thoth-tech/<repo-name>.git
+```
+
+#### Example with `splashkit.io-starlight` Repository
+
+```shell
+git remote add upstream https://github.com/thoth-tech/splashkit.io-starlight.git
+```
+
+### 3. Verify Upstream Branches
+
+Verify that the upstream branches have been added successfully.
+
+```shell
+git remote -v
+```
+
+### 4. Push Your Branch
+
+When you are ready to push your branch to create a pull request, push it to the `thoth-tech/splashkit.io-starlight` repository.
+
+### 5. Open GitHub to Review the Pull Request
+
+Open GitHub and navigate to the repository to review your pull request. Initially, the page will look like this:
+
+![pull request](/splashkit/pull-request-fig1.png)
+
+### 6. Change to the Correct Repository
+
+To ensure your pull request is directed to the correct repository, change the settings at the top of the page.
+
+- **Base Repository**: Set this to `thoth-tech/splashkit.io-starlight`.
+- **Base Branch**: Set this to `main`.
+
+![pull request](/splashkit/pull-request-fig2.png)
+
+![pull request](/splashkit/pull-request-fig3.png)
+
+### 7. Add Pull Request Template
+
+Use the pull request template to provide detailed information about your request. Make sure to fill out all necessary fields and complete any required checks.
+
+![pull request](/splashkit/pull-request-fig4.png)
+
+### 8. Submit Your Pull Request
+
+Once you have filled out the template and completed all required checks, submit your pull request for review. After submitted, move the associated planner card task into the first peer review and share the pull request and card into the group chat. Well done and keep an eye out for the feedback from your peer reviewer!
```

Comment:
This is difficult to read, recommend reviewing sentence structure.

_Suggestion:_
_After filling out the template and completing all required checks, submit your pull request for review. Once submitted, move the associated Planner card to the "First Peer Review" column and share the pull request and card in the group chat. Great job! Now, keep an eye out for feedback from your peer reviewer._

Reply:
fixed.

Reply:
> This is difficult to read, recommend reviewing sentence structure.
> 
> _Suggestion:_ _After filling out the template and completing all required checks, submit your pull request for review. Once submitted, move the associated Planner card to the "First Peer Review" column and share the pull request and card in the group chat. Great job! Now, keep an eye out for feedback from your peer reviewer._

Reading this thread is ass. Why're you still reviewing following an outline of your reviewed points? Just collate your points into one clear and concise comment for each file and pair it with your outline. No one wants to come here to read your process of reviewing the pull. Or are you prepared to review the 22 other requests?

Reply:
The review process is meant to collaborative, not combative. The language is not appropriate nor appreciated.

Reply:
> The review process is meant to collaborative, not combative. The language is not appropriate nor appreciated.

Cry abt it this is a public repo. If it was truly a "collaborative" experience you wouldn't make 50+ reviews for 1 pull.